Tests/LibWeb/Text/expected/localStorage.txt

```diff
@@ -1,6 +1,10 @@
+undefined
```

Comment:
Commit message: please add a space after "LibWeb:"

Reply:
Hopefully this is good :pray:

Reply:
Sure, but the commit message could still be a bit clearer. Maybe something like this?

> LibWeb/Storage: Return undefined for non-existent key/index access

Reply:
Let me know if you need the commit message changed again :+1: